src/pages/resultsView/pathwayMapper/ResultsViewPathwayMapper.tsx

```diff
@@ -202,6 +204,12 @@ export default class ResultsViewPathwayMapper extends React.Component<
                                 tabReflectsOql={true}
                             />
 
+                            <PathwayMapperMessageBox
+                                message={this.message}
+                                DEFAULT_MESSAGE={this.DEFAULT_MESSAGE}
+                                LOADING_MESSAGE={this.LOADING_MESSAGE}
```

Comment:
If you declare these as `const` variables outside this class you don't need to pass them to the `PathwayMapperMessageBox` component. If you still want to keep them anyway, then you should rename them to `defaultMessage` and `loadingMessage`.